cmake/Modules/LocalAom.cmake

```diff
@@ -23,6 +23,27 @@ if(EXISTS "${LIB_FILENAME}")
         # ext/avm/aom/aom_encoder.h includes config/aom_config.h which is generated by the local build of avm.
         target_include_directories(aom INTERFACE "${AOM_EXT_SOURCE_DIR}/build.libavif")
     endif()
+
+    # Add link dependency flags from the aom.pc file in ext/aom or ext/avm
+    # by prepending the build directory to PKG_CONFIG_PATH and then calling
+    # pkg_check_modules
+    if(WIN32)
+        set(ENV{PKG_CONFIG_PATH} "${AOM_EXT_SOURCE_DIR}/build.libavif;$ENV{PKG_CONFIG_PATH}")
+    else()
+        set(ENV{PKG_CONFIG_PATH} "${AOM_EXT_SOURCE_DIR}/build.libavif:$ENV{PKG_CONFIG_PATH}")
+    endif()
+
+    pkg_check_modules(_AOM QUIET aom)
+
+    set(_AOM_PC_LIBRARIES ${_AOM_STATIC_LIBRARIES})
```

Comment:
Do we need to quote `${_AOM_STATIC_LIBRARIES}`? Please compare this with line 46 in Findaom.cmake.

Reply:
I don't think it matters, but FindPkgConfig.cmake uses quotes in similar circumstances, so I'll change this to likewise be quoted, for consistency.